content/spin/v2/install.md

```diff
@@ -212,8 +212,10 @@ If you have [`cargo`](https://doc.rust-lang.org/cargo/getting-started/installati
 <!-- @selectiveCpy -->
 
 ```bash
-$ git clone https://github.com/fermyon/spin -b v2.0.0
+$ git clone https://github.com/fermyon/spin
 $ cd spin
+$ # check out the latest tagged release
+$ git checkout $(git tag --sort=taggerdate | tail -1)
```

Comment:
I don't think this will work on Windows.  I guess we could just put a comment saying "on Windows, find the latest tag from the GH releases page"?  Hard to show both without making selective copy mad though.  I guess we could use a tab block but it all starts to get a bit dreary then...!

Reply:
I think you can start a line with# and selective copy should be happy iirc.

Reply:
Updated with revised wording.  What do you all think?

As a follow-up, we could add automation so that on any Spin release, a PR is triggered in this repo to bump hard-coded versions in this doc (or any docs).  We could then return to `git clone https://github.com/fermyon/spin -b <tag>`.  Overkill?